src/Symfony/Bundle/FrameworkBundle/Controller/AbstractController.php

```diff
@@ -267,21 +268,33 @@ protected function render(string $view, array $parameters = [], Response $respon
     }
 
     /**
-     * Renders a view for a form.
+     * Renders a view and sets the appropriate status code when a form is listed in parameters.
      *
-     * The FormView instance is passed to the template in a variable named
-     * "form" (can be changed via $formVar argument).
-     * If the form is invalid, a 422 status code is returned.
+     * If an invalid form is found in the list of parameters, a 422 status code is returned.
      */
-    protected function renderForm(string $view, FormInterface $form, array $parameters = [], Response $response = null, string $formVar = 'form'): Response
+    protected function renderForm(string $view, array $parameters = [], Response $response = null): Response
     {
-        $response = $this->render($view, [$formVar => $form->createView()] + $parameters, $response);
+        if (null === $response) {
+            $response = new Response();
+        }
+
+        foreach ($parameters as $k => $v) {
```

Comment:
I think it could be better to pass explicitly the form object since it can save us from cycling all the parameters. 
```php
<?php
    protected function renderForm(string $view, FormInterface $form, array $parameters = [], Response $response = null): Response
```

Moreover, it improves simplest cases: instead of calling `$this->renderForm('view.html.twig', ['form' => $form)` you can use `this->renderForm('view.html.twig', $form)`

Reply:
@garak this was discussed on other PR. 
There are 2 problems with this approach: 
1. It makes less obvious what is variable name of form in view.
2. Doesn't work when have more than 1 form. (Typical use-case: Profile editing: 1 form for simple fields (name), second for for phone change, 3rd form for email change.

Reply:
That's exactly what this PR iterates over, see patch. Iterating makes the helper compatible with controllers that have multiple forms as parameters, and saves from creating any convention to name that $form parameter inside the list of $parameters (see removed $formVar)